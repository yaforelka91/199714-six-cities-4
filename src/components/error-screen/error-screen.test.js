import React from 'react';
import renderer from 'react-test-renderer';
import ErrorScreen from './error-screen.jsx';

describe(`ErrorScreenSnapshot`, () => {
  it(`should render ErrorScreen`, () => {
    const tree = renderer.create(
        <ErrorScreen message='Some error message' />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
