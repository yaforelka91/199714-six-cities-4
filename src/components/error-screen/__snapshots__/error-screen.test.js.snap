// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreenSnapshot should render ErrorScreen 1`] = `
<section>
  <div
    style={
      Object {
        "fontSize": "24",
        "textAlign": "center",
      }
    }
  >
    <b>
      Sorry, something went wrong :( Please, come back later
    </b>
    <p>
      It happens because 
      Some error message
    </p>
  </div>
</section>
`;
